packages/react-composites/src/composites/CallComposite/hooks/useAdaptedSelector.ts

```diff
@@ -84,7 +84,9 @@ const memoizeState = memoizeOne(
     callsEnded: [],
     deviceManager,
     callAgent: { displayName },
-    calls
+    calls,
+    // TODO: FIXME
```

Comment:
nit: why this comment?

Reply:
Removed in a PR higher in the stack. I added more context to the comment so it's not as enigmatic :D